skimage/registration/_phase_cross_correlation.py

```diff
@@ -143,6 +211,12 @@ def phase_cross_correlation(reference_image, moving_image, *,
         data will bypass FFT of input data. Case insensitive. Not
         used if any of ``reference_mask`` or ``moving_mask`` is not
         None.
+    disambiguate : bool
+        The shifts returned by this function are only accurate *modulo* the
+        image shape, due to the periodic nature of the Fourier transform. If
+        this is set to ``True``, the *real* space cross-correlation is
+        computed for each possible shift, and the shift with the highest
```

Comment:
I prefer 'shift' in the singular form, but so far you've used the plural, so we should be consistent one way or the other. I prefer 'shift' so that, in this context (in these docstrings), by 'shift' we mean the tuple, say, `(-150, 200)`, and by 'shifts' we mean the set of:

    (-150, 200)
    (-150, -1800)
    (1850, 200)
    (1850, -1800)

Otherwise, 'shifts' mean either `(-150, 200)` or the set of all possible shifts.